guile/guile-gnutls/Portfile

```diff
@@ -13,13 +14,18 @@ checksums           rmd160  4ac751aca5616735001f630951fdd0942d9a73c4 \
                     sha256  5b4cb926032076ec346bb5c0bc0d0231f968fe0f565913cc16934bb793afb239 \
                     size    747952
 
+gitlab.setup        gnutls guile ${version} v
```

Comment:
please add `version` here directly, no need to set it as a variable first and then use it here.

Reply:
again, if the gitlab PG works ad, for example, the github PG then doing this should JustWork. I'm afraid though that there are some discrepancies between the two and I am clearly not aware of those, that's were my comments originate from...